DeAceroDLabs/marketplace-frontend
# [DMP-632] Hide product until data is loaded

## What is done

[[DMP-632] Hide Product details until data is loaded](https://deacero.atlassian.net/browse/DMP-632)

## How is done

1. Added a checkpoint in the product detail to verify that the product detail is loaded before displaying it.

## How to test

1. Run the project via `npm start`
2. click on any product in the catalog section.
3. You should be redirected to the product page.
4. You won't see anything until the data is loaded.

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
